server/startup/accounts.js

```diff
@@ -229,9 +229,9 @@ export default function () {
       // in current version currentSessionId will be available for anonymous
       // users only, because it is unknown for me how to pass sessionId when user
       // logged in
-      const currentSessionId = options.methodArguments &&
-        options.methodArguments.length === 1 &&
-        options.methodArguments[0].sessionId;
+
+      // currentSessionId should be a string but the former declaration assign currentSessionId a boolean value.
```

Comment:
Generally we don't need comments in code to explain why a change was made. That's for commit messages and PR notes. This comment won't mean anything to anybody trying to understand this code.